src/browser-lib/parcel.ts

```diff
@@ -0,0 +1,131 @@
+import { Component, OnInit, OnDestroy, Input, OnChanges } from '@angular/core';
+
+@Component({
+  selector: 'parcel',
+  template: '<div>Parcel works!</div>'
+})
+export class ParcelComponent implements OnInit, OnDestroy, OnChanges {
+  @Input() config: any; // import { ParcelConfig } from 'single-spa';
+  @Input() mountParcel: any;
+  @Input() onParcelMount: () => void;
+
+  @Input() handleError = err => console.error(err);
+
+  @Input() wrapWith = "div";
+  @Input() customProps: any;
+  @Input() appendTo: any;
+
+
+  createdDomElement: any;
+
+  hasError: boolean;
+
+  unmounted: any;
+  nextThingToDo: Promise<any>;
+
+  title = 'utente';
+
+  parcel: any;
```

Comment:
We could use https://github.com/CanopyTax/single-spa/blob/master/typings/single-spa.d.ts#L21 here instead of any

Reply:
Good idea!

Reply:
Here isn't better a ParcelConfig ? The mountParcel accept parcelConfig and parcelConfigObject (
 https://github.com/CanopyTax/single-spa/blob/master/typings/single-spa.d.ts#L10 )

Reply:
Fixed! Missing update function!